src/app/lib/config/services/urdu.js

```diff
@@ -1,111 +1,110 @@
 import { C_POSTBOX } from '@bbc/psammead-styles/colours';
-import { latin } from '@bbc/gel-foundations/scripts';
+import { arabic } from '@bbc/gel-foundations/scripts';
 import { news as brandSVG } from '@bbc/psammead-assets/svgs';
 
 const service = {
-  lang: `ur-IN`,
-  articleAuthor: `https://www.facebook.com/bbcurdu`,
-  articleTimestampPrefix: 'Updated',
+  lang: 'ur',
+  product: 'BBC News',
+  articleAuthor: 'https://www.facebook.com/bbcnews',
+  articleTimestampPrefix: 'اپ ڈیٹ کی گئی',
   atiAnalyticsAppName: 'news-urdu',
   brandName: 'BBC News اردو',
-  product: 'BBC News اردو',
+  serviceLocalizedName: 'اردو',
   defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png',
   defaultImageAltText: 'BBC News اردو',
-  dir: `ltr`,
-  externalLinkText: ', external',
-  imageCaptionOffscreenText: 'Image caption, ',
-  videoCaptionOffscreenText: 'Video caption, ',
-  audioCaptionOffscreenText: 'Audio caption',
-  defaultCaptionOffscreenText: 'Caption, ',
-  imageCopyrightOffscreenText: 'Image source, ',
-  locale: `ur-IN`,
-  datetimeLocale: `ur-IN`.toLowerCase(),
+  dir: 'rtl',
+  externalLinkText: 'بیرونی مواد,',
+  imageCaptionOffscreenText: ' ,تصویر کا کیپشن',
+  videoCaptionOffscreenText: ' ,ویڈیو کیپشن',
+  audioCaptionOffscreenText: ' ,آڈیو کیپشن',
+  defaultCaptionOffscreenText: ' کیپشن',
+  imageCopyrightOffscreenText: ' ,تصویر کا ذریعہ',
+  locale: 'ur',
+  datetimeLocale: 'ur',
   service: 'urdu',
-  serviceName: 'News اردو',
+  serviceName: 'Urdu',
   themeColor: `${C_POSTBOX}`,
   twitterCreator: '@bbcurdu',
   twitterSite: '@bbcurdu',
   noBylinesPolicy: 'https://www.bbc.com/news/help-41670342#authorexpertise',
   publishingPrinciples: 'https://www.bbc.com/news/help-41670342',
-  script: latin,
+  script: arabic,
   manifestPath: '/articles/manifest.json',
   swPath: '/articles/sw.js',
   translations: {
     error: {
       404: {
         statusCode: '404',
-        title: '404 - صفہ دستیاب نہیں',
+        title: 'صفحہ دستیاب نہیں',
         message:
-          'ہو سکتا ہے آپ نے ویب سائٹ کا پتہ غلط درج کیا ہو۔ برائے مہربانی پتہ اور املا کی غلطیاں ٹھیک کریں',
+          'ہم معذرت خواہ ہیں کہ آپ جو صفحہ تلاش کر رہے ہیں وہ دستیاب نہیں۔ یہ کوشش کر کے دیکھیں:',
         solutions: [
-          'Double checking the url',
-          'Hitting the refresh button in your browser',
-          'Searching for this page using the BBC search bar',
+          'یو آر ایل دوبارہ چیک کریں',
+          'اپنے براؤزر کو ریفریش کریں',
+          'بی بی سی کی سرچ بار کی مدد سے اس صفحے کی تلاش جاری',
         ],
-        callToActionFirst: 'Alternatively, please visit the ',
-        callToActionLinkText: 'بی بی سی صفحۂ اول',
+        callToActionFirst: ' متبادل کے طور پر بی بی سی نیوز',
+        callToActionLinkText: '.کے ہوم پیچ پر جائیں اردو',
         callToActionLast: '',
         callToActionLinkUrl: 'https://www.bbc.com/urdu',
       },
       500: {
         statusCode: '500',
```

Comment:
```suggestion
        statusCode: '۵۰۰',
```

Reply:
Resolving for this reason: https://github.com/bbc/simorgh/pull/2706#discussion_r307236205